hooks/usePlayerActions/getActivePlayerIndexSet.ts

```diff
@@ -0,0 +1,15 @@
+import { GameStreetAction } from "../../models/GameSituation";
+
+export default function getActivePlayerIndexSet(
+  gameStreetActions: GameStreetAction[]
+) {
+  const activePlayerSet = new Set<number>();
+
+  for (const streetAction of gameStreetActions) {
+    if (activePlayerSet.has(streetAction.playerIndex)) break;
+
+    activePlayerSet.add(streetAction.playerIndex);
+  }
+
+  return activePlayerSet;
```

Comment:
```suggestion
  return new Set(streetAction.map(({ playerIndex }) => playerIndex))
```

Reply:
If anything, we don't need this function?

Reply:
@axross Thanks for your comments, but it doesn't work. It returns Set of initial action player indexes. Maybe the name should be getInitialActivePlayerIndexSet.

Reply:
@axross You are right.